pkgs/development/coq-modules/coq-elpi/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, which, coq, ocaml, ocamlPackages }:
+
+let params = {
+  "8.10" = {
+    version = "master";
+    rev = "bc7134deba1aacc7ecd2f5d1032bdf05b125c568";
+    sha256 = "188avk9irwjsbs5ya4ka01mpk3vw4397kv2rmsncqrrrsa1pdddk";
+  };
+};
+  param = params."${coq.coq-version}";
+in
+
+stdenv.mkDerivation rec {
+  name = "coq${coq.coq-version}-elpi-${param.version}";
+
+  src = fetchFromGitHub {
+    owner = "LPCIC";
+    repo = "coq-elpi";
+    inherit (param) rev sha256;
+  };
+
+  nativeBuildInputs = [ which ];
+  buildInputs = [ocaml] ++ (with ocamlPackages; [findlib elpi]);
+
+  installFlags = "COQLIB=$(out)/lib/coq/${coq.coq-version}/";
+  propagatedBuildInputs = [ coq ];
+
+  meta = {
+    description = "Coq plugin embedding ELPI.";
+    maintainers = [ stdenv.lib.maintainers.vbgl ];
```

Comment:
Please ask before doing so. Usually the author of the package puts is own name here.

Reply:
Hi @vbgl! Would you like to be maintainer? What are the other options?